src/pilot/dell_nfv.py

```diff
@@ -218,13 +218,14 @@ def edit_environment_files(
                 if ovs_dpdk:
                     dpdk_nics = self.find_ifaces_by_keyword(nic_env_file,
                                                             'Dpdk')
+                    logger.info("DPDK-NICs >>" + str(dpdk_nics))
```

Comment:
Could you change to logger.debug or remove ?

Reply:
Resolved.